qipr_approver/approver/templates/approver/dashboard.html

```diff
@@ -10,27 +10,29 @@
     <table class="striped">
         <thead>
             <tr>
-                <th data-field="number">Number</th>
+                <th data-field="number">#</th>
                 <th data-field="title">Title</th>
-                <th data-field="edit_link">Edit Project</th>
                 <th data-field="role">Role</th>
-                <th data-field="delete">Delete</th>
                 <th data-field="Approval">Project Approval</th>
+                <th data-field="delete">Delete?</th>
             </tr>
         </thead>
         <tbody>
             {% for project in projects %}
                 <tr>
                     <td>{{ forloop.counter }}</td>
-                    <td class="project__title">{{ project.title }}</td>
-                    <td><a href="{% url 'approver:projects' project_id=project.pk %}">link</a></td>
+                    <td><a href="{% url 'approver:projects' project_id=project.pk %}" class="project__title">{{ project.title }}</td>
```

Comment:
Different lines for nested html tags. Also if I have to scroll to the right to see all the tag, thats bad!